lib/Controller/ChattyLLMController.php

```diff
@@ -708,7 +708,9 @@ public function checkMessageGenerationTask(int $taskId, int $sessionId): JSONRes
 				return new JSONResponse(['task_status' => $task->getstatus()], Http::STATUS_EXPECTATION_FAILED);
 			}
 		} elseif ($task->getstatus() === Task::STATUS_RUNNING || $task->getstatus() === Task::STATUS_SCHEDULED) {
-			return new JSONResponse(['task_status' => $task->getstatus()], Http::STATUS_EXPECTATION_FAILED);
+			$startTime = $task->getStartedAt() ?? time();
+			$slowPickup = $task->getScheduledAt() + 60 * 5 < $startTime;
```

Comment:
```suggestion
			$slowPickup = ($task->getScheduledAt() + (60 * 5)) < $startTime;
```
Even if everybody should know operand priorities, it's more readable with parenthesis IMO.